src/pages/postgraphile/plugins.md

```diff
@@ -168,4 +168,27 @@ An example of a PostGraphile server plugin is [@graphile/operation-hooks](https:
 - uses `cli:library:options` to convert these CLI options to library options
 - uses `postgraphile:options` to a) convert the library options into graphileBuildOptions (Graphile Engine plugin options), and b) load the OperationHooksPlugin
 
+### Inline tweaks via pluginHook
+
+If you are using postgraphile as a library (i.e. middleware) and you want to make a quick tweak to something using the server plugin hooks, you can do so in your main server.js file:
+
+```js
+
+/**
+ * This plugin override changes the branding piece of graphiql.
+ */
+const graphiqlBrandingTweak = {
+    ['postgraphile:graphiql:html'](html) {
+        console.log("Applying GraphiQL Branding Tweak...");
+        return html.replace('i.a.createElement("div",null,"PostGraph",i.a.createElement("em",null,"i"),"QL")', '"GraphiQL for MyCompany"');
```

Comment:
This example is a little too brittle, it depends on the minified source code which could change for any number of reasons. Is there a more stable change you could make, for example adding a CSS file before `</head>`?

Reply:
Yeah, I think we can use text transformation css for this example.  I'll work on that and update the PR.

Reply:
Thanks! 🙏